### PR TITLE
Configure RSpec and add tests for Letters Controller

### PR DESCRIPTION
**Pull request by rna**
## Summary
- Removed the Minitest boilerplate files
- Configured RSpec for the project
- Added tests for the Letters Controller

### CODE REVIEW

File: spec/controllers/letters_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  describe 'POST #create' do
+    context 'without authentication' do
+      it 'should redirect to login page' do
+        post :create, params
+        expect(response).to redirect_to  %r(\Ahttp://test.host/users/sign_in)
+      end
+    end
+
+    context 'with authentication' do
+    login_user
```
Comment (rtdp) [inline spec/controllers/letters_controller_spec.rb:14]: Indentation correction required.

File: spec/controllers/letters_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  end
+
+  describe 'POST #like' do
+    context 'without authentication' do
+      it 'should redirect to login page' do
+        letter = Letter.create(description: "Ruby is awesome")
+        post :like, params: {id: letter.id}
+        expect(response).to redirect_to  %r(\Ahttp://test.host/users/sign_in)
+      end
+    end
```
Comment (rtdp) [inline spec/controllers/letters_controller_spec.rb:95]: Need positive case test

**Review by rtdp — CHANGES_REQUESTED** [review]

File: spec/controllers/letters_controller_spec.rb (hunk 1 above)
Comment (rna) [reply spec/controllers/letters_controller_spec.rb]: Corrected.

File: spec/controllers/letters_controller_spec.rb (hunk 2 above)
Comment (rna) [reply spec/controllers/letters_controller_spec.rb:105]: added positive test case for like method.

**Pull request merged**